04-advanced-topics/network-analysis_rnaseq_01_wgcna.Rmd

```diff
@@ -286,6 +291,23 @@ df <- round(df) %>%
   dplyr::filter(rowSums(.) >= 50)
 ```
 
+Another thing we need to do is make sure our main experimental group label is set up. 
+In this case `refinebio_treatment` has two groups: `pre-adt` and `post-adt`. 
+We need this to be a factor, but we also need `pre-adt` as the first level. 
+
+```{r}
+metadata <- metadata %>%
+  dplyr::mutate(refinebio_treatment = factor(refinebio_treatment,
```

Comment:
This is a place where I might make a new variable name for simplicity, but I don't feel strongly about it.

Reply:
If you didn't feel strongly, I didn't do it.